CharacterCreation/07_Items.txt

```diff
@@ -1,165 +1,213 @@
+use device.
+
+Engineer Material Pack ($60):
+    Contains 80 material suitable for use with an engineer's 3D extruder tool.
+
+HD Zip-Cuffs ($300):
+    Pack of 5 carbon-fiber zipe-tie cuffs. Strength DC 95 or INT 105 to Break.
+
+Medic Needle Pack ($60):
+    Contains 10 needles suitable for use with several malpractice proceedures.
+Not recomended for medicianal use. 
 
-MediKit: $100
-For normal wounds (50% damage or less) roll percents under 50. On pass, heal 30
-+ percent rolled. For severe wounds (50% damage or more), roll percents under 
+MediKit ($100):
+    For normal wounds (50% damage or less) roll percents under 50. On pass, heal 
+30+ percent rolled. For severe wounds (50% damage or more), roll percents under 
 50. On pass, heal 30. If you have Skill in Medicine, see the skill sheet for 
 separate instructions.
 
-Repair kit: $100
-For normal damage (50% or less) roll percents under 50. On pass, repair 30 + 
+Glow Stick ($4):
+    A clear plastic tube with filled with a colored phosphorus compound that 
+emits enough light to see by in a 5m radius. When activated they make a 
+muffled crunching noise and then last for about two hours. They are available 
+in the following colors.
+*  Red
+*  Orange
+*  Gold
+*  Green
+*  Cyan
+*  White ($1 extra)
+*  Blacklight ($2 extra) 
+
+Nanite Pack ($100):
+    Restores 2d10 + 20 nanites.
+
+Permanent Marker ($2):
+    MarkAll(tm) permanent marking pen of one of the following colors.
+
+* Black
+* Red
+* Blue
+* Dark Green
+* Orange
+* Purple
+* Gold
+* Silver
+
+Repair kit ($100):
+    For normal damage (50% or less) roll percents under 50. On pass, repair 30 + 
 percent rolled. For severe wounds (50% or more), roll percents under 50. On 
-pass, repair 30.
-If you have Skill in Repair, see the skill sheet for separate instructions.
+pass, repair 30. If you have Skill in Repair, see the skill sheet for separate 
+instructions.
 
-Nanite Pack: $100
-Restores 2d10 + 20 nanites.
-
-Security Spike: $100
-+20 to a hack check on a door with a physical access panel. DC 30 to place. 
+Security Spike ($100):
+    +20 to a hack check on a door with a physical access panel. DC 30 to place. 
 Once hacked the door can be controlled remotely.
 
-Medic Needle Pack: $60
-Contains 10 needles suitable for use with several malpractice proceedures.
-Not recomended for medicianal use. 
-
-Single Use door brute-forcer $120:
-Hacks through the encryption on a door (up to 75 DC) and opens it.
-
 Zip-Cuffs ($25):
-pack of 5 nilon fiber woven standard zip-tie cuffs. Strength DC 70 or INT 80 
+    Pack of 5 nilon fiber woven standard zip-tie cuffs. Strength DC 70 or INT 80 
 to break.
 
-HD Zip-Cuffs ($300):
-Pack of 5 carbon-fiber zipe-tie cuffs. Strength DC 95 or INT 105 to Break.
-
 ==============================
 Explosives
 ==============================
-Breaching charges ($500):
-Will remove a door from your path. Make sure no one is within 2m of the door
+
+Breaching charges ($160):
```

Comment:
Did I change these prices? I'm trying to remember who did.

Reply:
`$git blame`
looks like most of the prices are from Trowl223's commits back in the day, but Anfo pricing was me:
https://github.com/bleehu/Compound_X/commit/721e21199383764db768aa017eb1441867a8b7a6

Reply:
Sounds good. I forgot I could look stuff up :P